examples/README.md

```diff
@@ -0,0 +1,24 @@
+# examples
+
+This folder contains some examples showing the usage of the Golang
+packages provided by Inspektor Gadget.
```

Comment:
You could add a paragraph explaining that the examples are for developers who want to reuse code from Inspektor Gadget. End-users don't need to use those: they can use local-gadget or kubectl-gadget instead.